neoforge-main/src/main/java/dev/compactmods/crafting/client/ClientEventHandler.java

```diff
@@ -82,4 +101,43 @@ private static void doFieldPreviewRender(RenderLevelStageEvent event, Minecraft
 //                });
         buffers.endBatch();
     }
+
+    @SubscribeEvent
+    public static void onRightClickBlock(final PlayerInteractEvent.RightClickBlock event) {
```

Comment:
Why are you using an event for this instead of the block methods? The proxies share a base class.